cli/README.md

```diff
@@ -16,22 +16,31 @@ executable.
 
 ## Installation
 
-To install Zinnia, you need to have Rust tooling installed on your machine. See
-[Install Rust](https://www.rust-lang.org/tools/install).
+You can download the `zinnia` binary from
+[our GitHub Releases](https://github.com/filecoin-station/zinnia/releases/latest).
+
+| OS      | Platform      | Filename                                                                                                               |
+| ------- | ------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| Windows | Intel, 64bit  | [zinnia-windows-x64.zip](https://github.com/filecoin-station/zinnia/releases/latest/download/zinnia-windows-x64.zip)   |
```

Comment:
The URL format is described here: https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases

(I learned about this today.)